processing/src/main/java/io/druid/query/filter/JavaScriptDimFilter.java

```diff
@@ -21,32 +21,36 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Joiner;
 import com.google.common.base.Preconditions;
 import com.metamx.common.StringUtils;
 
 import java.nio.ByteBuffer;
+import java.util.Arrays;
 
 public class JavaScriptDimFilter implements DimFilter
 {
-  private final String dimension;
+  private final String[] dimensions;
   private final String function;
 
   @JsonCreator
   public JavaScriptDimFilter(
+      // for backwards compatibility
       @JsonProperty("dimension") String dimension,
```

Comment:
This is going into 0.9.0 with other API changes so I think we can in this case

But would be good to get some other feedback here